core/claim_workflow.py

```
@@ -0,0 +1,480 @@
+            claim.approved_date = timezone.now()
+            claim.save()
+            
+            # Notify stakeholders
+            self.notifier.notify_claim_approved(claim_id)
+            
+            return {
+                'success': True,
+                'claim_id': claim_id,
+                'stage': ClaimWorkflowStage.APPROVAL.value,
+                'approved_by': approver_id
+            }
```

Ruff (0.13.2)

Consider moving this statement to an `else` block

(TRY300)